cfme/tests/configure/test_default_views_cloud.py

```diff
@@ -26,54 +25,56 @@
 # TODO refactor for setup_provider parametrization with new 'latest' tag
 
 
-def set_and_test_default_view(group_name, view, page):
-    old_default = DefaultView.get_default_view(group_name, fieldset='Clouds')
-    DefaultView.set_default_view(group_name, view, fieldset='Clouds')
+def set_and_test_default_view(appliance, group_name, view, page):
+    default_views = appliance.user.my_settings.default_views
+    old_default = default_views.get_default_view(group_name, fieldset='Clouds')
+    default_views.set_default_view(group_name, view, fieldset='Clouds')
     selected_view = navigate_to(page, 'All', use_resetter=False).toolbar.view_selector.selected
-    assert view == selected_view, "{} view setting failed".format(view)
-    DefaultView.set_default_view(group_name, old_default, fieldset='Clouds')
+    assert view == selected_view, '{} view setting failed'.format(view)
+    default_views.set_default_view(group_name, old_default, fieldset='Clouds')
 
 # BZ 1283118 written against 5.5 has a mix of what default views do and don't work on different
```

Comment:
Go ahead and delete this comment please, that BZ was marked duplicate and the duplicate was closed in 5.8